sherpa/astro/utils/tests/test_astro_utils_xspec.py

```diff
@@ -635,6 +635,83 @@ def __init__(self, name='abcd'):
     assert len(caplog.records) == 0
 
 
+@requires_xspec
+def test_create_model_multiplicative_xspec(caplog):
+    """Fake up a multplicative model for XSPEC"""
+
+    model = StringIO("""abcd           1  0.         1.e20           foos    mul  0
+nH      cm^-3   1.0   1.e-6  1.e-5  1.e19  1.e20   -0.01
+
+""")
+    parsed = xspec.parse_xspec_model_description(model)
+
+    converted = xspec.create_xspec_code(parsed, internal=(3, 4, 0))
+
+    expected = '''
+@version_at_least("3.4.0")
+class XSabcd(XSMultiplicativeModel):
+    """The XSPEC abcd model:  TBD
+
+    The model is described at [1]_.
+
+    .. versionadded:: ???
+       This model requires XSPEC 3.4.0 or later.
+
+    Parameters
+    ----------
+    nH
+
+    References
+    ----------
+
+    .. [1] https://heasarc.gsfc.nasa.gov/xanadu/xspec/manual/XSmodelAbcd.html
```

Comment:
This URL doesn't seem to work- returns "Sorry, but the document you requested is not available on this server"

Reply:
Yeah - that's by intent, as 

- this is for a test and it is just checking that the model name has been capitalized correctly (`abcd` -> `Abcd`)
- the output is for text that needs to be checked before it can be directly used, as - thanks to the way that XSPEC models are documented - we can not guarantee that `XSmodel<name>.html` is valid, but it's going to be close to being the truth